scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1198,8 +1220,58 @@ static s32 create_node(u32 node_id)
 			break;
 		}
 
-		if (*nmask & (1LLU << (cpu % 64)))
+		if (*nmask & (1LLU << (cpu % 64))) {
 			bpf_cpumask_set_cpu(cpu, cpumask);
+			if (!(cctx = lookup_cpu_ctx(-1))) {
+				scx_bpf_error("cpu ctx error");
+				ret = -ENOENT;
+				break;
+			}
+			cctx->node_idx = node_id;
+		}
+	}
+
+	bpf_rcu_read_unlock();
+	return ret;
+}
+
+static s32 create_cache(u32 cache_id)
```

Comment:
This method probably should have existed for awhile, but I hadn't had a use for the `cache_ctx` yet.